.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
@malaterre Astyle source code is C++ and apparently the fact that a project is c, c++ or both is declared in cmake at the PROJECT() scope, so as soon as you have C++ code you must enable C++. Is it really an issue to require a C++ compiler available ? I don't think so honestly.